src/core/components/courses/SlotContainer.vue

```diff
@@ -425,9 +425,9 @@ export default {
   padding: 0
   margin: 10px 0px
   &-subtitle
-     font-style: italic
-     font-size: 16px
-     @media screen and (max-width: 767px)
+    font-style: italic
+    font-size: 16px
+    @media screen and (max-width: 767px)
       font-size: 13px
```

Comment:
Tu sais pourquoi les `px` sont en rouges ?

Reply:
Non je n'ai pas trouvé ! C'est l'éditeur github qui considère qu'il y a une erreur mais je ne comprends pas pourquoi : https://stackoverflow.com/questions/31793515/css-property-is-red-highlighted-in-github